### PR TITLE
[FrontendV2] Add UI support for submission meta attributes

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR - 

- [x] Adds support for submission meta attributes on submit page in Frontend V2

Screenshot from 2021-12-19 18-47-29

### CODE REVIEW

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
@Ram81 Minor change.

**Comment by Ram81** [comment]
@RishabhJain2018 I don't see the comment for the change. Can you comment it again?

File: frontend_v2/src/app/services/global.service.ts [hunk 1]
```diff
@@ -333,6 +333,7 @@ export class GlobalService {
     let value = '';
     let valueFound = false;
     components.map((item) => {
+      console.log(item.label + " -- " + label + " ---- " + item.value);
```
Comment (RishabhJain2018) [inline frontend_v2/src/app/services/global.service.ts:336]: Please remove the console statement.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

**Comment by RishabhJain2018** [comment]
@Ram81 Please resolve the conflicts here.

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**